package.json

```diff
@@ -76,15 +77,12 @@
     "test:node:watch": "cd packages/airnode-node && yarn run test:watch",
     "test:validator": "(cd packages/airnode-validator && yarn run test)"
   },
-  "dependencies": {
-    "ethers": "^5.4.5",
```

Comment:
These are not needed for the root package

Reply:
Also some packages from `devDependencies` like:
* `@types/jest`
* `@types/lodash`
* `dotenv`
* `rimraf`